regression/goto-instrument/argc-argv2/test2.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--model-argc-argv 2
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
+^warning: ignoring
```

Comment:
This test only tests that the `--model-argc-argv 2` is accepted, I don't think it tests the actual value `2` is used anywhere (this file fails without the flag).